webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelect.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import TypeAheadSelect from '.';
+import {
+  props,
+  propsWithNew,
+  propsWithPlaceholder,
+  propsWithMultiple,
+  propsWithClearButton,
+} from './TypeAheadSelect.fixtures';
+import Story from '../../../../../..//stories/components/Story';
+import storeDecorator from '../../../../../..//stories/storeDecorator';
+
+export default {
+  title: 'Components|TypeAheadSelect',
+  decorators: [storeDecorator],
```

Comment:
Please add, `component: TypeAheadSelect` so it will render `prop-types-table` into the docs page.